wifi_password/wifi_password.py

```diff
@@ -113,25 +152,33 @@ def generate_qr_code(ssid: str, password: str, path: str, show_qr: bool) -> None
         # in order to display the QR code on Windows
         colorama.init()
         qr.make()
+        print_info('QR CODE:')
         qr.print_tty()
 
     if path:
-        file_name = ssid.replace(" ", "_") + ".png"
+        file_name = ssid.replace(" ", "_")
+        file_name = file_name.replace(".", "_")
 
         if path == "STORE_LOCALLY":
-            path = file_name
+            path = file_name + ".png"
 
```

Comment:
Revert it back to the way it was. Having periods throughout the file name is fine and the adding of ".png" should happen on the same line as where the `replace()` function is run.